lib/sles4sap_publiccloud.pm

```diff
@@ -129,23 +136,28 @@ sub get_promoted_hostname {
 
     Clean up Network peering and qesap deployment
 
+=over 3
```

Comment:
This is with arguments in capital letters

![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/101880010/414a8bae-00bd-4b0f-af99-7ed638c01707)


This is with lowercase

![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/101880010/d4732e6c-6a6d-404b-b3c9-5f1db9d2e4c3)

Reply:
I decided to change all of them in this file in lowercase, I'll probably do the same in `lib/qesapdeployment.pm` soon or later